script/getchannels.py

```diff
@@ -93,7 +95,18 @@ def getNodesOrigin(channel):
     break
   index += channels['count']
 
-with open('../public/data/channelslist.json', "w") as file_write:
+# Récupération du chemin ou est installé le script
+scriptFileName = pprint(inspect.getfile(inspect.currentframe())) 
+# scriptFileName contiens habituellement le chemin complet mais pas toujours 
+# (ce n'est pas le cas sur Mac OSX par exemple).
+scriptFolderPath = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
On doit pouvoir utiliser ça plutôt 
`scriptFolderPath = os.path.dirname(os.path.abspath(__file__))`

comme c'est indiqué là : https://stackoverflow.com/questions/3430372/how-to-get-full-path-of-current-files-directory-in-python

Reply:
Oui je confirme ca fait pareil chez les mangeurs de pommes. J'ai changé.

```
mbp-b12:script b12$ cat test__file__.py 
#!/usr/bin/python

import os 
import inspect

print('__file__ : '+ os.path.dirname(os.path.abspath(__file__)))
print('inspect  : '+ os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))))

mbp-b12:script b12$ ./test__file__.py 
__file__ : /Users/b12/CF/git/crazy-toads.github.io/script
inspect  : /Users/b12/CF/git/crazy-toads.github.io/script
```